README.md

```diff
@@ -73,7 +74,7 @@ Plug 'SirVer/ultisnips' | Plug 'honza/vim-snippets'
 Plug 'scrooloose/nerdtree', { 'on':  'NERDTreeToggle' }
 Plug 'tpope/vim-fireplace', { 'for': 'clojure' }
 
-" Using git URL
+" Any valid git URL is allowed
 Plug 'https://github.com/junegunn/vim-github-dashboard.git'
```

Comment:
The only one that I know was `http://git.wincent.com/command-t.git` but even this one has a mirror on Github.

(did you really wanted to ping me? or it was for junegunn? :stuck_out_tongue_closed_eyes: )